packages/kit/src/views/Market/MarketHomeV2/utils.ts

```
@@ -0,0 +1,57 @@
+// Shared utility functions for MarketHomeV2 components
+
+/**
+ * Parse a string value to number, supporting K (thousands) and M (millions) suffixes
+ * @param value - String value like "10K", "5M", "1000"
+ * @returns Parsed number value
+ */
+export const parseValueToNumber = (value: string): number => {
+  const cleanValue = value.replace(/[^0-9.]/g, '');
+  const numValue = parseFloat(cleanValue);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)